WIP SM-110 removing voice commands

JeffHowes opened the pull request.

JeffHowes committed:
starting WIP PR

JeffHowes committed:
Improved removeCommand() and added to voice demo "unmount"

JeffHowes committed:
Used Jacob's version of removeCommand

toha87 commented on src/hooks/useSpeechRecognition.js:
```diff
@@ -130,6 +130,17 @@ const useSpeechRecognition = () => {
     };
 
+    const removeCommand = (command) => {
+        let commandIndex = intendArray.indexOf(command);
```
Will this search by the reference or value?

JeffHowes committed:
removed unneeded useSpeechSynthesis instance

JeffHowes committed:
Added debugging to the remove command

JeffHowes committed:
Added a console log for the IntendArray for troubleshooting

JeffHowes committed:
Stopped voice commands from repeating 

Added a if CommandFound var check.  We still want to remove commands when we leave a page. This is a partial fix to the bug.

JeffHowes commented:
Deleting this pull request, as this solution did not fix the problem
